roles/ucp/templates/ucp.j2

```diff
@@ -40,12 +40,20 @@ start)
     {% else -%}
     /usr/bin/docker run --rm -t --name ucp \
         -v /var/run/docker.sock:/var/run/docker.sock \
-        -e UCP_ADMIN_USER="admin" -e UCP_ADMIN_PASSWORD="orca" \
+        -e UCP_ADMIN_USER={{ ucp_admin_user }} -e UCP_ADMIN_PASSWORD={{ ucp_admin_password }} \
         docker/ucp join --host-address={{ node_addr }} \
           --image-version={{ ucp_version }} \
           --url="https://{{ service_vip }}:443" \
+    {% if ucp_replicator1_node is defined and node_addr == ucp_replicator1_node_addr -%}
```

Comment:
@mapuri sure, `run_as == master` works as long as we are sure that we want all master to be replicas.. I will make the change